src/app/shell/personal-cabinet/provider/create-workshop/create-description-form/create-description-form.component.ts

```diff
@@ -145,4 +163,46 @@ export class CreateDescriptionFormComponent implements OnInit, OnDestroy {
   setFocus(): void {
     this.keyWordsInputElement.nativeElement.focus();
   }
+
+  /**
+   * This method creates new FormGroup
+   */
+  private newForm(item?: WorkshopSectionItem): FormGroup {
+    const EditFormGroup = this.formBuilder.group({
+      sectionName: new FormControl('', [Validators.required]),
+      description: new FormControl('', [
+        Validators.required,
+        Validators.minLength(ValidationConstants.INPUT_LENGTH_1),
+        Validators.maxLength(ValidationConstants.MAX_DESCRIPTION_LENGTH_2000),
+      ]),
+    });
+
+    if (item) {
+      EditFormGroup.addControl(
+        'workshopId',
+        this.formBuilder.control(item.workshopId)
+      );
+      EditFormGroup.patchValue(item, { emitEvent: false });
```

Comment:
follow the approach of adding new form controls as it is in this PR. So we won't have any problems with dirty forms in the future https://github.com/ita-social-projects/OoS-Frontend/pull/1308